test/redeemer.ts

```diff
@@ -62,22 +70,116 @@ describe("Redeemer", () => {
     assert.isOk(proposalId);
     assert.isOk(result.votingMachine);
 
+    const gp = (await dao.getSchemes("GenesisProtocol"))[0].wrapper as GenesisProtocolWrapper;
+    await helpers.vote(result.votingMachine, proposalId, BinaryVoteResult.No, accounts[2]);
+    await helpers.vote(result.votingMachine, proposalId, BinaryVoteResult.Yes, accounts[3]);
+
+    await (await gp.stakeWithApproval({ amount: web3.toWei(10), vote: 1, proposalId })).getTxMined();
+
     await helpers.vote(result.votingMachine, proposalId, BinaryVoteResult.Yes, accounts[1]);
 
-    await helpers.increaseTime(1);
+    await helpers.increaseTime(259200);
```

Comment:
add a comment here that this is the boosting timeout default value.

Reply:
done